### PR TITLE
AB#104298 equalities - add cypress test

### PR DESCRIPTION
**Pull request by jordan-beard**
### Checklist

- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    afterEach(() => {
+        cy.storeSessionData();
+    });
+
+    beforeEach(function () {
+        cy.visit(`${Cypress.env('url')}/project/10002121/benefits`);
+    });
+
+    it('Should save yes', () => {
+        cy.get('a[data-test=equalities-impact-assessment]').click();
```
Comment (Lisa-V-Sau) [inline end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js:13]: You have the character 'a' in the parameter, is that a typo?

File: end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        cy.get('[data-test=true]').click();
+        cy.get('[data-test=submit-btn]').click();
+
+        // check Yes has been saved
+        cy.get('p[data-test=equalities-impact-assessment]').should($el => expect($el.text()).to.equal('Yes'));
```
Comment (Lisa-V-Sau) [inline end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js:18]: Again here.

File: end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+        // back to task list
+        cy.get('[href="/project/10002121"]').click();
+
+        // check template preview        
+        cy.get('[data-test=preview-htb]').click();
+        cy.get('p[data-test=equalities-impact-assessment]').should($el => expect($el.text()).to.equal('Yes'));
```
Comment (Lisa-V-Sau) [inline end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js:25]: Again here.

File: end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+        // check change link
+        cy.get('a[data-test=equalities-impact-assessment]').click();
```
Comment (Lisa-V-Sau) [inline end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js:28]: Again here.

File: end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        cy.get('[data-test=header]').should($el => expect($el.text().trim()).to.equal(`Bristol And South Gloucestershire Utc\n                            \n                            Has an Equalities Impact Assessment been considered?`));
+    });
+
+    it('Should save no', () => {
+        cy.get('a[data-test=equalities-impact-assessment]').click();
```
Comment (Lisa-V-Sau) [inline end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js:33]: Again here.

File: end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        cy.get('[data-test=false]').click();
+        cy.get('[data-test=submit-btn]').click();
+
+        // check No has been saved
+        cy.get('p[data-test=equalities-impact-assessment]').should($el => expect($el.text()).to.equal('No'));
```
Comment (Lisa-V-Sau) [inline end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js:38]: Again here.

File: end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+        // back to task list
+        cy.get('[href="/project/10002121"]').click();
+
+        // check template preview        
+        cy.get('[data-test=preview-htb]').click();
+        cy.get('p[data-test=equalities-impact-assessment]').should($el => expect($el.text()).to.equal('No'));
```
Comment (Lisa-V-Sau) [inline end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js:45]: Again here.

File: end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js (hunk 1 above)
Comment (jordan-beard) [reply end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js:13]: No - there were two elements with this data attribute and the easiest way to distinguish was to look for an anchor element with this data attribute.

File: end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js (hunk 7 above)
Comment (jordan-beard) [reply end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js:45]: No - there were two elements with this data attribute and the easiest way to distinguish was to look for an paragraph element with this data attribute.

**Review by Lisa-V-Sau — COMMENTED** [review]
Just a few comments from me but overall looks good. Thanks Jordan!

**Review by Lisa-V-Sau — APPROVED** [review]

**Pull request merged**